02-oop-inheritance/javaScript/refactored_JS/downloableMovie v0.1.js

```diff
@@ -0,0 +1,94 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function ( title ) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function ( paramName, paramValue ) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
```

Comment:
I thought the exercise as it is in the practice. Notice that it says (attributes: hashMap). 
Of course, I could've used plain objects as attributes. (E.g.: title, duration, mainActor, etc;)
But I wanted the model and the exercise to be alike.
I can model it otherwise , if you prefer